core/includes/bootstrap.inc

```diff
@@ -3258,7 +3258,13 @@ function _backdrop_bootstrap_sanitize_request() {
     }
 
     // If there is a query string, check its query parameters.
-    $destination_parts = backdrop_parse_url($_GET['destination']);
+    if (isset($_GET['destination'])) {
+      $destination_parts = backdrop_parse_url($_GET['destination']);
+    }
+    else if (isset($_REQUEST['destination'])) {
+      $destination_parts = backdrop_parse_url($_REQUEST['destination']);
+    }
```

Comment:
The addition of `$_REQUEST['destination']` seems like a change in functionality? Can we remove this `else if` statement?

Reply:
Also a few lines below, looks like we need to check if each exists when doing:
```
      unset($_GET['destination']);
      unset($_REQUEST['destination']);
```

Even though that triggers a fatal error, it shouldn't also cause a PHP warning.